hw_config.h

```diff
@@ -970,6 +970,36 @@
 # define BOARD_USART_PIN_CLOCK_BIT      RCC_AHB1ENR_GPIODEN
 # define SERIAL_BREAK_DETECT_DISABLED   1
 
+/****************************************************************************
+ * TARGET_HW_SMARTAP_PRO
+ ****************************************************************************/
+
+#elif  defined(TARGET_HW_SMARTAP_PRO)
+
+# define APP_LOAD_ADDRESS               0x08008000
+# define BOOTLOADER_DELAY               5000
+# define BOARD_SMARTAP_PRO
+# define INTERFACE_USB                  1
+# define INTERFACE_USART                0
+# define USBDEVICESTRING                "SmartAP PRO BL"
```

Comment:
You will need to add the prefix "PX4 BL" or retool the uploader 

https://github.com/PX4/Bootloader/blob/f7a97e3a776bd386495c4b771e07555ad5688315/hw_config.h#L11-L18